clients/lightrider/lightrider-entry.js

```diff
@@ -163,4 +163,7 @@ if (typeof window !== 'undefined') {
 
 export {
   runLighthouseInLR,
+  lighthouse,
```

Comment:
can the above manual window assignment be removed? does umd bundle assign to global for us in browser context?

Reply:
the window assignment can be removed if we switch our LR client from `window.runLighthouseInLR` to `window.lightriderBundle.runLighthouseInLR`.  
(aka: doable)

Reply:
`name` is necessary for umd, and also is used to namespace on the global. makes sense.

That seems like a decent change to make IMO. Although, I would just name it `lightrider`